### PR TITLE
GateWatcher: Push the AIONIQ format

### PR DESCRIPTION
**Pull request by squioc**
Push the AIONIQ format in the repository

### CODE REVIEW

File: GateWatcher/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GateWatcher
+
+## Description
+Gatewatcher is a cybersecurity software provider specializing in advanced intrusion detection and is a market leader in high-performance solutions based on automation and machine learning methods.
```
Comment (PierrickV) [inline GateWatcher/README.md:4]:
```suggestion
Gatewatcher is a cybersecurity software provider specializing in advanced intrusion detection and in high-performance solutions based on automation and machine learning methods. 
```

This sentence looks too promotional

File: GateWatcher/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# GateWatcher
+
+## Description
+Gatewatcher is a cybersecurity software provider specializing in advanced intrusion detection and is a market leader in high-performance solutions based on automation and machine learning methods. 
+
+## Intakes
+*
```
Comment (PierrickV) [inline GateWatcher/README.md:7]:
```suggestion
```

We could remove this part from `cookiecutter`

File: GateWatcher/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# GateWatcher
```
Comment (PierrickV) [inline GateWatcher/README.md:1]: The logo file should be:
- max 50 KB (+57.6 KB for this one)
- transparent (I don't think that this one is)

File: GateWatcher/aioniq/_meta/fields.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
+gatewatcher.type:
+    description: Type of analysis
+    name: gatewatcher.type
+    type: keyword
+
+gatewatcher.flow_id:
+    description: identifier of the flow
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:7]:
```suggestion
    description: Identifier of the flow
```

File: GateWatcher/aioniq/_meta/fields.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    name: gatewatcher.flow_id
+    type: keyword
+
+gatewatcher.event_type:
+    description: type of event
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:12]:
```suggestion
    description: Type of event
```

File: GateWatcher/aioniq/_meta/fields.yml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    name: gatewatcher.event_type
+    type: keyword
+
+gatewatcher.malcore.code:
+    description: return code of the malcore analysis
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:17]:
```suggestion
    description: Return code of the malcore analysis
```

File: GateWatcher/aioniq/_meta/fields.yml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    name: gatewatcher.malcore.code
+    type: keyword
+
+
+gatewatcher.malcore.detail_threat_found:
+    description: type of the detected threat
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:23]:
```suggestion
    description: Type of the detected threat
```

File: GateWatcher/aioniq/_meta/fields.yml [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    name: gatewatcher.malcore.detail_threat_found
+    type: keyword
+
+gatewatcher.malcore.file:
+    description: identifier of the file
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:28]:
```suggestion
    description: Identifier of the file
```

File: GateWatcher/aioniq/_meta/fields.yml [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    name: gatewatcher.timestamp_analyzed
+    type: keyword
+
+gatewatcher.timestamp_detected:
+    description: Timestamp of the file collection by gcap
+    name: gatewatcher.timestamp_detected
+    type: keyword
+
+gatewatcher.codebreaker.file_id:
+    description: identifier of file
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:88]:
```suggestion
    description: Identifier of file
```

File: GateWatcher/aioniq/_meta/manifest.yml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+uuid: bba2bed2-d925-440f-a0ce-dbcae04eaf26
+name: AIONIQ
+slug: aioniq
+
+description: >-
+  A new detection and response platform (NDR) that enables to identify with certainty malicious actions and suspicious behaviors based on a mapping of all assets present on the information system.
+
+data_sources:
+  Network intrusion detection system:
+  Network protocol analysis:
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/manifest.yml:10]: The sentences are empty here

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 11]
```diff
@@ -0,0 +1 @@
+~/Documents/Travail/Projets/InThreat/sekoia.io/intake-formats/GateWatcher/aioniq/_meta
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/smart-descriptions.json:1]:
```suggestion
[]
```

**Review by PierrickV — COMMENTED** [review]
Thank you for this work :smiley:

File: GateWatcher/aioniq/_meta/fields.yml [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+    description: Type of event
+    name: gatewatcher.event_type
+    type: keyword
+
+gatewatcher.malcore.code:
+    description: Return code of the malcore analysis
+    name: gatewatcher.malcore.code
+    type: keyword
+
+
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:21]:
```suggestion

```

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 13]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/smart-descriptions.json:1]: This part is empty

File: GateWatcher/aioniq/tests/suricata_alert_event.json [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+    "related": {
+      "hosts": [
+        "network.internal"
+      ],
+      "ip": [
+        "9.8.7.6"
+      ]
+    },
+    "rule": {
+      "category": "Potesekoiaally Bad Traffic",
```
Comment (PierrickV) [inline GateWatcher/aioniq/tests/suricata_alert_event.json:48]:
```suggestion
      "category": "Potentially Bad Traffic",
```

You have a strange typo here

File: GateWatcher/aioniq/tests/suricata_alert_event.json [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
.......T)].f..._.I.X0K.k...|-t...\\\\.d#7.A..J..I.L.H7:.r..%].Ti......(....V-i....2...:...`J...\\\"S\\\"..?I.......w..E....Q.......B.l$.T.E....-......k.u........BQ.#.Tn@.C..x.7.K/...M...},..-L.......~..E.@..o.7.. .!.t....._q.....\\\\........H...Y...MA...`U.8..O..z.J.l#91..\\\".+...Vi..v..k......%.k...0i..u.T.O#A.[j.M...*G*W..s.......V..+.%.......t:..&<....Uz..2.....{....\\\\.{a.H.-.D.QC..]|>3..t5.........9.._n.U..1Ly.....(v.Fm...agn..zs.s=0..........;..U..\\n.........bs...[={.A....oG...7.../.}...yz.>......7......B;.....m\\r.../....F!../O./.n...~~..u$.~....hz..e..n.@(.=.Ui.../.\\\\_-F{..........W....~...g}......W........uWvm..ve1~n...vo_<.....=.......}e.v..gOl.^D{vJ..k_........>......y|.........k.=..W.?}.s.../^......=.4.#=.~..l?.}.}k._.....K>...k....._...:...N........`}C......w.................:.wW...Z.....~.....}.._..%?.W8.....$.R..y...............sCq.....y.....)^e....gS^<z..G...|.G....\\n)p,.|...v7.............LMY._.o.......y.......\",\"packet_info\":{\"linktype\":1},\"in_iface\":\"monvirt\",\"dest_ip\":\"1.2.3.4\",\"timestamp_analyzed\":\"2022-06-03T14:59:41.373Z\",\"gcap\":\"gcap-sekoia.gatewatcher.com\",\"tx_id\":0,\"uuid\":\"525084c9-9a40-4dc9-81fb-27d5efe6b965\",\"severity\":2}"
```
Comment (PierrickV) [inline GateWatcher/aioniq/tests/suricata_alert_event.json:9]:
```suggestion
    "message": "{\"@timestamp\":\"2022-06-03T14:59:41.373Z\",\"gcenter\":[\"gcenter-sekoia.gatewatcher.com\",\"gcenter-sekoia.gatewatcher.com\"],\"event_type\":\"alert\",\"payload\":\"SFRUUC8xLjEgMjAwIE9LCkRhdGU6IFRodSwgMDIgSnVuIDIwMjIgMjI6Mzc6MjIgR01UClNlcnZlcjogQXBhY2hlLzIuNC40MSAoVWJ1bnR1KQpMYXN0LU1vZGlmaWVkOiBUaHUsIDAyIEp1biAyMDIyIDIyOjMwOjM0IEdNVApFVGFnOiAiMTdkOS01ZTA3ZThkZGI0NTA4LWd6aXAiCkFjY2VwdC1SYW5nZXM6IGJ5dGVzClZhcnk6IEFjY2VwdC1FbmNvZGluZwpDb250ZW50LUVuY29kaW5nOiBnemlwCkNvbnRlbnQtTGVuZ3RoOiAyNDg1CktlZXAtQWxpdmU6IHRpbWVvdXQ9NSwgbWF4PTEwMApDb25uZWN0aW9uOiBLZWVwLUFsaXZlCkNvbnRlbnQtVHlwZTogdGV4dC9odG1sCgp0ZXN0Cg==\",\"packet\":\"CAAnjitsCAAnk+hwCABFAAAoBRhAAD8GMWkKAQHewKg4yABQwow7Z24SQI3k4FAQAfUWzAAA\",\"type\":\"suricata\",\"community_id\":\"1:dGVzdAo=\",\"app_proto\":\"http\",\"src_ip\":\"9.8.7.6\",\"dest_port\":49804,\"alert\":{\"action\":\"allowed\",\"rev\":2,\"signature\":\"ETPRO INFO Observed Suspicious Base64 Encoded Wide String Inbound (exe)\",\"category\":\"Potentially Bad Traffic\",\"gid\":1,\"metadata\":{\"updated_at\":[\"2020_11_17\"],\"created_at\":[\"2020_04_13\"],\"former_category\":[\"HUNTING\"],\"signature_severity\":[\"Informational\"],\"attack_target\":[\"Client_Endpoint\"],\"deployment\":[\"Perimeter\"],\"affected_product\":[\"Windows_XP_Vista_7_8_10_Server_32_64_Bit\"]},\"signature_id\":2841990,\"severity\":2},\"flow\":{\"pkts_toserver\":5,\"bytes_toserver\":798,\"start\":\"2022-06-03T14:59:08.750205+0000\",\"pkts_toclient\":4,\"bytes_toclient\":3052},\"files\":[{\"filename\":\"/exploit.html\",\"state\":\"CLOSED\",\"tx_id\":0,\"sid\":[1100029],\"magic\":\"HTML document, ASCII text, with very long lines\",\"gaps\":false,\"md5\":\"16e3fcee85f81ec9e9c75dd13fb08c01\",\"sha256\":\"2c36fbcbac3e57df410f6613180fe572015adba62d0f1bd98c13a1535d64703c\",\"size\":6105,\"stored\":false}],\"proto\":\"TCP\",\"stream\":1,\"host\":\"network.internal\",\"http\":{\"protocol\":\"HTTP/1.1\",\"hostname\":\"www.xmlformats.com\",\"http_content_type\":\"text/html\",\"length\":2485,\"http_user_agent\":\"Mozilla/4.0 (compatible; ms-office; MSOffice 16)\",\"http_method\":\"GET\",\"url\":\"/exploit.html\",\"status\":200},\"timestamp_detected\":\"2022-06-03T14:59:08.780Z\",\"ether\":{\"src_mac\":\"08:00:27:8e:2b:6c\",\"dest_mac\":\"08:00:27:93:e8:70\"},\"src_port\":80,\"flow_id\":1686930575880829,\"payload_printable\":\"HTTP/1.1 200 OK\\r\\nDate: Thu, 02 Jun 2022 22:37:22 GMT\\r\\nServer: Apache/2.4.41 (Ubuntu)\\r\\nLast-Modified: Thu, 02 Jun 2022 22:30:34 GMT\\r\\nETag: \\\"17d9-5e07e8ddb4508-gzip\\\"\\r\\nAccept-Ranges: bytes\\r\\nVary: Accept-Encoding\\r\\nContent-Encoding: gzip\\r\\nContent-Length: 2485\\r\\nKeep-Alive: timeout=5, max=100\\r\\nConnection: Keep-Alive\\r\\nContent-Type: text/html\\r\\n\\r\\n...........Xko........\\n.F&.$VS..]pmYRa.Vd9q.(.........gW......#7....G....s.=.RO.....q..&n.....0.k...|{D.....!6.....V&nB.6.oVap......}7........l..>..{>{..~k.n..f.5]o.....X..k._G....U.....|...\\\\.a.m.f......._.!...c.8.Z..n.0........i..`.:..c[.a..;......_.........gv}.L.1V.G.......o.2,}..C~..w.(,...[..at+..8.~..'.mh1a..y......hVc0.n.iB.en.Z..O.]...l.b..2.b..{|i|._+...o].3}..Wd....3\\\"...!:.............C./.Z.....\\rP$S,.t<Y..m.E.]5Y-...Sx.A..1...[W.@.......kKlb...m.3..n./......c...\\n..@y0.....5.........$ .#..|\\r.......;.w}....`.)..u....U^.....lD...D.#...&...jaT.........@@..Lf..6l.z........p>.s.k..!..r..UI..g...ji^V...,.k..0i...}.!.=.......2.%.@..=u........{'Y@.k.8!.*`... ..c..z.j.u.D.....*......G.ng.U.....@.3U......\\n...$/..!.c.....T..S..tr.$...h......$(....&R...i.U#PL.J{...\\n!E.-9,w.....$%Xh9.U!...6...S`b...C>.i.cW......H...It\\n...B......q.IR....\\n..P&....i.d... .07.]U$tD.R...J4............^....tIT....UaD....g..k.b.......\\rm.VcK....p:....P.Dj...\\nD*0u*..b..(..P...\\\\S..Q*VT'......m.............7B..D./\\\"...gX..\\\".9W....I.=.9......T.%.U....J{b.l.\\r..Q.X.t9U.i)......R.i..V.g.5c..^.,.....&=r..p0SX..E...S5hsSJt..J...'}#8.........R.H.D.(i.TW...^.&..>@v..+sX\\ra..],>I.!%.`l`..,vDvL.....vDwM....,.I.-[3IP.I..GMi.I.MYa..'Z$U]r...... j3CE).NM!.@.!a......T.S.77....k&...P.........8...$..:.A.....+A........a......Mm..*..\\\\..zZ\\\"\\n...D.I.e.....r..9..JD..8.u`vd{..=.)Y.9...\\\\A'.}J...'.A?....)...........U....M5.`....J.&..e.D....N{1.s...d....cZE....\\nG)..8.nq)..G..`..@.T.rgB..B.9>7.@.\\\\&#'EUT...;Xt?...P.%W'.,@(\\r.+Y...4.y~.{d.&xn\\\"...../].....k.m.ZK`..M.lr.....VK.\\\"z&.R+.V.<-..U.\\\"...IU.h%/9....y....T)].f..._.I.X0K.k...|-t...\\\\.d#7.A..J..I.L.H7:.r..%].Ti......(....V-i....2...:...`J...\\\"S\\\"..?I.......w..E....Q.......B.l$.T.E....-......k.u........BQ.#.Tn@.C..x.7.K/...M...},..-L.......~..E.@..o.7.. .!.t....._q.....\\\\........H...Y...MA...`U.8..O..z.J.l#91..\\\".+...Vi..v..k......%.k...0i..u.T.O#A.[j.M...*G*W..s.......V..+.%.......t:..&<....Uz..2.....{....\\\\.{a.H.-.D.QC..]|>3..t5.........9.._n.U..1Ly.....(v.Fm...agn..zs.s=0..........;..U..\\n.........bs...[={.A....oG...7.../.}...yz.>......7......B;.....m\\r.../....F!../O./.n...~~..u$.~....hz..e..n.@(.=.Ui.../.\\\\_-F{..........W....~...g}......W........uWvm..ve1~n...vo_<.....=.......}e.v..gOl.^D{vJ..k_........>......y|.........k.=..W.?}.s.../^......=.4.#=.~..l?.}.}k._.....K>...k....._...:...N........`}C......w.................:.wW...Z.....~.....}.._..%?.W8.....$.R..y...............sCq.....y.....)^e....gS^<z..G...|.G....\\n)p,.|...v7.............LMY._.o.......y.......\",\"packet_info\":{\"linktype\":1},\"in_iface\":\"monvirt\",\"dest_ip\":\"1.2.3.4\",\"timestamp_analyzed\":\"2022-06-03T14:59:41.373Z\",\"gcap\":\"gcap-sekoia.gatewatcher.com\",\"tx_id\":0,\"uuid\":\"525084c9-9a40-4dc9-81fb-27d5efe6b965\",\"severity\":2}"
```

**Comment by PierrickV** [comment]
Parent task:
- https://github.com/SekoiaLab/platform/issues/30695

File: GateWatcher/aioniq/tests/suricata_alert_event.json (hunk 14 above)
Comment (squioc) [reply GateWatcher/aioniq/tests/suricata_alert_event.json]: Fixed. Thanks

File: GateWatcher/aioniq/_meta/fields.yml [hunk 16]
```diff
@@ -0,0 +1,84 @@
+gatewatcher.type:
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/fields.yml:1]: I wonder if this field could be an [observer.type](https://www.elastic.co/guide/en/ecs/current/ecs-observer.html#field-observer-type)

**Review by PierrickV — APPROVED** [review]

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 17]
```diff
@@ -0,0 +1,80 @@
+[{
+    "value": "Block infected payload hosted on {destination.address}: {gatewatcher.malcore.detail_threat_found}",
```
[the remark this answers is not in the document]
Comment (squioc) [reply GateWatcher/aioniq/_meta/smart-descriptions.json]: Fixed

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+        },
+        {
+            "field": "gatewatcher.malcore.detail_threat_found"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "source.address",
+            "target": "destination.address",
+            "type": "connection was denied"
```
[the remark this answers is not in the document]
Comment (squioc) [reply GateWatcher/aioniq/_meta/smart-descriptions.json]: changed

File: GateWatcher/aioniq/_meta/fields.yml (hunk 16 above)
Comment (squioc) [reply GateWatcher/aioniq/_meta/fields.yml:1]: The semantic of `observer.type` seems different from `gatewatcher.type`

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 19]
```diff
@@ -0,0 +1,91 @@
+[{
+    "value": "Infected payload {file.name} hosted on {url.domain} was blocked: {gatewatcher.malcore.detail_threat_found}",
+    "conditions": [
+        {
+            "field": "gatewatcher.type",
+            "value": "malcore"
+        },
+        {
+            "field": "state",
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/smart-descriptions.json:9]:
```suggestion
            "field": "gatewatcher.state",
```

I am not sure that only `"state"` will work

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+        },
+        {
+            "field": "event.severity"
+        },
+        {
+            "field": "rule.name"
+        }
+    ]
+},{
+    "value": "{file.name} from {url.domain} analyzed",
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/smart-descriptions.json:52]:
```suggestion
    "value": "{file.name} originating from {url.domain} analyzed",
```

File: GateWatcher/aioniq/_meta/smart-descriptions.json [hunk 21]
```diff
@@ -0,0 +1,91 @@
...
+        },
+        {
+            "field": "url.domain"
+        },
+        {
+            "field": "file.name"
+        }
+    ]
+},{
+    "value": "Connection from {source.address} to {url.domain}",
```
Comment (PierrickV) [inline GateWatcher/aioniq/_meta/smart-descriptions.json:66]:
```suggestion
    "value": "{gatewatcher.event_type} connection from {source.address} to {url.domain}",
```

Improvement suggestion :smiley: 

It's always going to be `http` but the user can click on it to filter/search this value

File: GateWatcher/aioniq/ingest/parser.yml [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+          event.category: ['malware']
+          event.type: ['info']
+
+  suricata:
+    actions:
+      - set:
+          source.bytes: '{{json_load.message.flow.bytes_toclient}}'
+          source.packets: '{{json_load.message.flow.pkts_toclient}}'
+          destination.bytes: '{{json_load.message.flow.bytes_toserver}}'
+          destination.packets: '{{json_load.message.flow.pkts_toserver}}'
```
Comment (PierrickV) [inline GateWatcher/aioniq/ingest/parser.yml:85]:
```suggestion          
          source.bytes: '{{json_load.message.flow.bytes_toserver}}'
          source.packets: '{{json_load.message.flow.pkts_toserver}}'
          destination.bytes: '{{json_load.message.flow.bytes_toclient}}'
          destination.packets: '{{json_load.message.flow.pkts_toclient}}'
```

You might have inverted these fields e.g.:

> [destination.bytes](https://www.elastic.co/guide/en/ecs/current/ecs-destination.html#field-destination-bytes): Bytes sent from the destination to the source.

File: GateWatcher/aioniq/ingest/parser.yml [hunk 23]
```diff
@@ -0,0 +1,112 @@
...
+          rule.version: '{{json_load.message.alert.rule.version}}'
+          rule.category: '{{json_load.message.alert.category}}'
+          event.action: '{{json_load.message.alert.action}}'
+        filter: '{{json_load.message.alert != null}}'
+      - set:
+          file.name: '{{json_load.message.fileinfo.filename}}'
+          file.size: '{{json_load.message.fileinfo.size}}'
+          file.hash.md5: '{{json_load.message.fileinfo.md5}}'
+          file.hash.sha256: '{{json_load.message.fileinfo.sha256}}'
+        filter: '{{json_load.message.fileinfo != null}}'
```
Comment (PierrickV) [inline GateWatcher/aioniq/ingest/parser.yml:112]:
```suggestion

          url.path: '{{json_load.message.http.url}}'
          url.domain: '{{json_load.message.http.hostname}}'
          http.version: '{{json_load.message.http.version}}'
          http.request.method: '{{json_load.message.http.http_method}}'
          user_agent.original: '{{json_load.message.http.http_user_agent}}'
          http.response.status_code: '{{json_load.message.http.status}}'

          dns.id: '{{json_load.message.dns.id}}'
          dns.type: '{{json_load.message.dns.type}}'
          dns.question.name: '{{json_load.message.dns.rrname}}'
          dns.question.type: '{{json_load.message.dns.rrtype}}'

          rule.name: '{{json_load.message.alert.signature}}'
          rule.id: '{{json_load.message.alert.signature_id|string}}'
          rule.version: '{{json_load.message.alert.rule.version}}'
          rule.category: '{{json_load.message.alert.category}}'
          event.action: '{{json_load.message.alert.action}}'

          file.name: '{{json_load.message.fileinfo.filename}}'
          file.size: '{{json_load.message.fileinfo.size}}'
          file.hash.md5: '{{json_load.message.fileinfo.md5}}'
          file.hash.sha256: '{{json_load.message.fileinfo.sha256}}'
```

You could simplify the code here

**Review by PierrickV — COMMENTED** [review]
Thank you for this work! :smiley_cat:

File: GateWatcher/aioniq/_meta/smart-descriptions.json (hunk 20 above)
Comment (squioc) [reply GateWatcher/aioniq/_meta/smart-descriptions.json]: Applied

File: GateWatcher/aioniq/_meta/smart-descriptions.json (hunk 19 above)
Comment (squioc) [reply GateWatcher/aioniq/_meta/smart-descriptions.json]: fixed

File: GateWatcher/aioniq/ingest/parser.yml (hunk 22 above)
Comment (squioc) [reply GateWatcher/aioniq/ingest/parser.yml]: fixed

File: GateWatcher/aioniq/ingest/parser.yml (hunk 23 above)
Comment (squioc) [reply GateWatcher/aioniq/ingest/parser.yml]: refactored

File: GateWatcher/aioniq/_meta/smart-descriptions.json (hunk 21 above)
Comment (squioc) [reply GateWatcher/aioniq/_meta/smart-descriptions.json]: Fixed

**Review by PierrickV — APPROVED** [review]

**Pull request merged**